content/applications/productivity/iot/devices/screen.rst

```diff
@@ -46,14 +48,15 @@ screen.
 .. image:: screen/screen-pos-client-display-no-order.png
    :align: center
    :alt: The default "POS Client Display" screen that appears when a screen display is successfully
-         connected to an IoT Box.
+         connected to an IoT box.
 
 The screen should also appear in the list of :guilabel:`Displays` on the :abbr:`IoT (Internet of
-Things)` Box homepage.
+Things)` box home page. Alternatively the display can also be seen by accessing :menuselection:`IoT
+app --> Devices`.
 
 .. image:: screen/screen-screen-name-example.png
    :align: center
-   :alt: An example of a screen display name shown on the IoT Box homepage.
+   :alt: An example of a screen display name shown on the IoT box home page.
 
 .. note::
    If no screen is detected, a default display named :guilabel:`Distant Display` will be used
```

Comment:
Can we give any more info about what the "Distant Display" is? Does this just mean there is no screen connected?

Reply:
Yes, this indicates that there is no hardware screen connected.